doc/DEVELOPMENT.org

```diff
@@ -17,6 +17,7 @@ I'm in the process of putting up better developer's documentation, but you shoul
    :         --publish # [optional], release to the Chrome Web Store/Mozilla addons
 
    You'll find the result in =dist/=. After that, you can load it in your browser and develop.
+   For firefox, note that mozilla intentionally complicates this in several ways. On android, only firefox nightly will let you install arbitrary extensions. On desktop, you have a choice between running nightly, re-installing your unpacked extension after every firefox restart, or setting up signing through web-ext. For a quick start, on desktop, chrome is therefore recommended (and is the author's primary browser, i believe?). On mobile, firefox nightly is your only option (?).
```

Comment:
> and is the author's primary browser

Nope, my primary browser is Firefox! Yeah, it's very annoying indeed that temporary extensions don't survive the restart, but I'm actually using the released version from addons.mozilla.org myself! So using your own build of the extension is hardly a 'quick start' since it's awailable from chrome/mozilla stores :)

And yeah.. for phone it used to work, but then new Firefox happened and now we're waiting till they add extension support back (I hope!). However since covid happened, I'm not using my phone that much either so wasn't too bothered about it so far enough to install Firefox nightly.